docs/tools.rst

```diff
@@ -58,9 +58,9 @@ You can use the Aiven platform in the way that best fits your workflow with our
         :shadow: md
         :margin: 2 2 0 0
 
-        **Aiven Terraform Provider** 
+        **Aiven Provider for Terraform** 
```

Comment:
nit: I think the original wording is what people google

Reply:
this is the official product name